mkosi

```diff
@@ -1301,13 +1302,44 @@ def load_defaults(args):
     config.optionxform = str
     config.read_file(f)
 
+    # this is used only for validation
```

Comment:
If the section/key name are valid, so that if we encounter and invalid section or key name the file name containing it can be printed.

Basically All files are read up in a options dict, and  then it is used in **load_defaults** to set values in the real args object.